### ISSUE TITLE
Entityquery assumes node entity

### ISSUE THREAD

**Issue by matslats** [amitaibu/og#112]
in \Drupal\og\Plugin\EntityReferenceSelection\OgSelection::buildEntityQuery line 79
$query->condition('type', $bundles, 'IN');
'type' is assumed to be the bundle key of the node entity.
So the following removes that assumption, starting from line 77

```
if ($bundle_key = \Drupal::entityTypeManager()->getDefinition($target_type)->getKey('bundle')) {  
  $bundles = Og::groupManager()->getAllGroupBundles($target_type);
  $query->condition($bundle_key, $bundles, 'IN');
}
```

**Comment by damiankloip** [comment]
I don't think it's assuming node. More things than node use type. I agree
that should be dynamic though.

**Comment by amitaibu** [comment]
@matslats  nice catch. Care to create a PR? :)

**Comment by damiankloip** [comment]
Yeah! The snippet of code in the summary looks just fine.

**Comment by matslats** [comment]
I need a bit of help creating PRs. I have done one but I have a whole queue here and only know how to push one commit at a time and compare amitai's HEAD with my HEAD

**Comment by amitaibu** [comment]
@matslats you can create a branch on your repo per issue. Maybe [this](https://help.github.com/articles/using-pull-requests/) can help.

**Comment by matslats** [comment]
Ah yes I missed the bit about branching the first time. So if I've got 4 changes do I have to make 4 branches locally? That means when I make a local change I put it on a new branch and I can only use one branch at a time!
Can I delete or merge my local branch after I pushed it to git hub and made a diff? 
It all seems like if it do it right, it is 5 times more work than copying and pasting and if I do it wrong I can lose hours! 
Amitai Would you refuse low tech change suggestions as I started to do before? I can make them very clear and easy to deal with!

**Comment by pfrenssen** [comment]
This was fixed in #132.
